app/assets/stylesheets/stages_nav.scss

```
@@ -0,0 +1,12 @@
+ol a:nth-of-type(1){
```

Aqui talvez seja melhor adicionarmos um 
```suggestion
.stage-bar ol a:nth-of-type(1){
```

Pois dessa maneira atual acredito que vai ser aplicado esse comportamento em qualquer `ol` que aparece no site